### DISCUSSION TITLE
confused ABOUT ORDER OF MULTIPLE FACE SWAPPING

### DISCUSSION THREAD

**Discussion by Thanh1122kkll** [glucauze/sd-webui-faceswaplab#156] [General]
I'm confused because there isn't any guide for swapping faces of multiple people. I've counted from left to right and matched the faces accurately, but after 10 swaps, it still doesn't align correctly. Is there a specific numbering required for 'target face'? In my case, there are two rows - two people in the top row and two in the bottom row. Should the numbering start from 0 in the top or bottom row?

**Comment by TexNorwegian** [comment]
Target Face 1 should be numbered 0
Target Face 2 should be numbered 1
Target Face 3 should be numbered 2
Target Face 4 should be numbered 3

and so on. You can change how many Targets you want in the FSL settings. It's near the top in the settings.....

_Max faces units (requires restart)_

I have mine set at 6.
